src/sentry/api/serializers/models/apitoken.py

```diff
@@ -27,6 +27,13 @@ def serialize(self, obj, attrs, user):
             "state": attrs.get("state"),
         }
         if not attrs["application"]:
-            data["token"] = obj.token
+            include_token = kwargs.get("include_token", False)
+            if include_token:
+                data["token"] = obj.token
+
             data["refreshToken"] = obj.refresh_token
+
+        data["tokenLastCharacters"] = (
+            obj.token_last_characters if obj.token_last_characters else obj.token[:4]
+        )
```

Comment:
We should not need to do this. Update the logic for the create endpoint to automatically add this value, and then create a backfill to update all empty columns with their respective values

Reply:
Captured in ticket issue https://github.com/orgs/getsentry/projects/163/views/1?pane=issue&itemId=47892760

Reply:
if this field doesn't exist yet, will checking for it error?

Reply:
It's actually already part of the table: https://github.com/getsentry/sentry/blob/984e550f261d34a2cf86a15b535df332711d1385/src/sentry/models/apitoken.py#L42

Reply:
Just realized I'm grabbing the first 4 characters instead of the last 4, should be `[-4:]`, let me update